### DISCUSSION TITLE
#33. VATZ biweekly meeting at 2023-05-10 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#432] [General]
### 1. Overall

<img width="1335" alt="image" src="https://user-images.githubusercontent.com/6308023/236984997-b190a2ad-60a4-4104-9698-92117b23e3d1.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
31 |  2/18 | 11
32 |  2/17 | 10
---

<img width="1353" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/54b37dcd-5062-47dc-90d6-a1f4eda4aa4d">

**Comment by xellos00** [comment]
# 동요

- #321
- #323 
- #388
- #397
- #435

---
@dsrvlabs/validator 
- https://github.com/dsrvlabs/vatz/issues/391#issuecomment-1541030584

@skonhwang 
- https://github.com/dsrvlabs/vatz/issues/326#issuecomment-1541168311

@gnongs 
- https://github.com/dsrvlabs/vatz/issues/344#issuecomment-1540946657
- https://github.com/dsrvlabs/vatz/issues/329#issuecomment-1541180716
- https://github.com/dsrvlabs/vatz/issues/417

@kim201212 
- https://github.com/dsrvlabs/vatz/issues/434
- https://github.com/dsrvlabs/vatz/issues/333#issuecomment-1541178133
- https://github.com/dsrvlabs/vatz/issues/339#issuecomment-1541185143

**Comment by rootwarp** [comment]
# 준교
- #426

**Comment by heejin-github** [comment]
## 희진
- #423 
  - https://github.com/dsrvlabs/vatz-plugin-sysutil/pull/9

**Comment by gnongs** [comment]
## 진홍
- #417

**Comment by skonhwang** [comment]
# 승곤
- no issue

**Comment by kim201212** [comment]
## 형기

- no issue

**Comment by meetrick** [comment]
## 황제
- no issue
